README.md

```diff
@@ -18,6 +19,103 @@ Specifically, if it enables services to scale _during_ deployment.
 > [!TIP]
 > A dashboard combining CloudFormation events, ASG activity and ASG metrics is available here - https://metrics.gutools.co.uk/d/cdvsv1d6vhp1cb/testing-asg-rolling-update.
 
+## Summary
+### Does `AutoScalingRollingUpdate` enable scaling events during deployment?
+No.
+
+When `AutoScalingRollingUpdate` is used, the CloudFormation update will remain in the `UPDATE_IN_PROGRESS` state 
+until the correct number of `SUCCESS` signals have been received.
+
+If more `SUCCESS` signals are received (due to a scale-out mid-deployment) the CloudFormation update will happily continue.
+
+If fewer `SUCCESS` signals are received (due to a scale-in mid-deployment) the CloudFormation update _could_ fail.
+There is no guarantee which instance will be terminated, with the the docs stating:
+
+> Amazon EC2 Auto Scaling features such as instance maintenance policies, termination policies, 
+> and scale-in protection are not available for use with CloudFormation rolling updates. 
+> Plan your rolling updates accordingly.
+> 
+> – https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-updatepolicy.html#cfn-attributes-updatepolicy-rollingupdate
+
+If a newly launched instance is terminated, the CloudFormation update will never receive the required amount of `SUCCESS` signals and a rollback will begin.
+
+For this reason, we recommend suspending the `AlarmNotification` process when using `AutoScalingRollingUpdate`,
+else availability of a service cannot be guaranteed.
```

Comment:
If in the future AWS support suspending scale-in and scale-out independently, then we should be able to scale-out during deployment, and scale-in post-deployment.